resources/dicts/relationship_events/normal_interactions/admiration/decrease.json

```diff
@@ -1,152 +1,215 @@
+		],
+		"reaction_random_cat": {
+			"admiration": "decrease"
+		},
+		"main_status_constraint": [
+			"elder",
+			"apprentice",
+			"warrior",
+			"mediator apprentice",
+			"mediator",
+			"medicine cat apprentice",
+			"medicine cat",
+			"deputy",
+			"leader"
+		],
+		"random_status_constraint": [
+			"elder",
+			"apprentice",
+			"warrior",
+			"mediator apprentice",
+			"mediator",
+			"medicine cat apprentice",
+			"medicine cat",
+			"deputy",
+			"leader"
+		]
+	},
+	{
+		"id": "admire_adult_de_med1",
+		"interactions": [
+			"m_c saw r_c miss an easy catch today.",
+			"m_c is disappointed in r_c's subpar work lately.",
+			"m_c is let down by r_c's sloppy performance during a sparring session.",
+			"r_c was supposed to be showing m_c a cool battle move but messed up so badly it made m_c cringe.",
+			"m_c noticed that r_c ignored an easy hunting opportunity on a recent patrol.",
+			"r_c froze up during a hunting patrol with m_c.",
+			"When m_c's border patrol encountered another clan, r_c acted like a fool.",
+			"m_c and r_c have very different approaches to hunting and argue about which is better."
+		],
+		"main_status_constraint": [
+			"apprentice",
+			"warrior",
+			"deputy",
+			"leader"
+		],
+		"random_status_constraint": [
+			"apprentice",
+			"warrior",
+			"deputy",
+			"leader"
+		]
+	},
+	{
+		"id": "admire_adult_de_med2",
+		"interactions": [
+			"m_c has noticed r_c isn't doing a lot of work lately.",
+			"m_c is disappointed in r_c's lack of initiative.",
+			"m_c notices that r_c seems to disappear whenever difficult tasks are being discussed.",
+			"r_c was meant to take a piece of prey to someone sick, but m_c saw {PRONOUN/r_c/object} eat it.",
+			"r_c didn't finish an important chore that m_c assigned {PRONOUN/m_c/object}.",
+			"The Clan notices that m_c and r_c won't talk to each other after coming back from a patrol."
+		],
+		"main_status_constraint": [
+			"elder",
+			"apprentice",
+			"warrior",
+			"deputy",
+			"leader",
+			"medicine cat",
+			"mediator"
+		],
+		"random_status_constraint": [
+			"apprentice",
+			"warrior",
+			"mediator apprentice",
+			"mediator",
+			"medicine cat apprentice",
+			"medicine cat",
+			"deputy",
+			"leader"
+		]
+	},
+	{
+		"id": "admire_de_high1",
+		"intensity": "high",
+		"interactions": [
+			"m_c heard an ugly rumor about r_c.",
+			"m_c overheard r_c spreading a hurtful rumor.",
+			"r_c won't stop complaining to m_c!",
+			"m_c thinks r_c did something incompetent.",
+			"m_c is embarrassed to be seen around r_c.",
+			"m_c learns about a terrible mistake r_c did.",
+			"m_c realizes r_c stole something from {PRONOUN/m_c/poss} nest.",
+			"m_c overhears r_c say something awful about a dead Clanmate.",
+			"m_c struggles to remember the last time r_c did something helpful.",
+			"When a bout of whitecough spreads around camp, m_c notices that r_c doesn't take it seriously.",

```

Comment:
Should be removed as we can't guarantee that any whitecough is spreading when this appears.